### ISSUE TITLE
Issue with boolean values provided in `bindings`

### ISSUE THREAD

**Issue by tk-o** [TroyAlford/react-jsx-parser#84]
This issue is really similar to that one https://github.com/TroyAlford/react-jsx-parser/issues/83

Looks like the boolean values are not evaluated correctly.

```
jsx="(display 1: {true ? 1 : 0}); (display 0: {false ? 1 : 0})"
```

**Actual:**
```
(display 1: ); (display 0: )
```

**Expected:**
```
(display 1: 1); (display 0: 0)
```

**Comment by TroyAlford** [comment]
I actually suspect the issue is not properly evaluating ternaries. When I run an example like this one in my sandbox, it looks like the `{false ? 1 : 0}` part simply returns `undefined`. I'm guessing this is because the Ternary itself is not implemented.

If you'd like to take a shot at it - I'd welcome a PR. Otherwise, I'll try to get to this sometime soon. It's a totally reasonable bit of syntax to be able to use.

**Comment by TroyAlford** [comment]
Closed by #85
